container/sagemaker/proxy_client.py

```diff
@@ -73,6 +74,36 @@ def add_model(self, model_name, base_path, model_platform='tensorflow'):
 
         return 'Successfully loaded model {}'.format(model_name)
 
+    def delete_model(self, model_name):
+        model_server_config = model_server_config_pb2.ModelServerConfig()
+
+        with lock(DEFAULT_LOCK_FILE):
+            try:
+                config_file = self._read_model_config(MODEL_CONFIG_FILE)
+                model_server_config = text_format.Parse(text=config_file,
+                                                        message=model_server_config)
+
+                config_json = json_format.MessageToJson(model_server_config)
```

Comment:
why convert to json and then dict instead of just working with pb2 objects?

Reply:
Because the request only provides `model_name` and `model_server_config` pb2 is not subscriptable. Converting the config list to json makes it easier to locate the model config that is going to be deleted.

Reply:
not subscriptable, but iterable: https://developers.google.com/protocol-buffers/docs/reference/python-generated#repeated-message-fields

Reply:
Used pb2 message to modify model config.